server/src/test/java/com/broadcom/lsp/cobol/service/delegates/completions/SectionCompletionTest.java

```diff
@@ -0,0 +1,77 @@
+package com.broadcom.lsp.cobol.service.delegates.completions;
+
+import com.broadcom.lsp.cobol.service.CobolDocumentModel;
+import com.broadcom.lsp.cobol.service.delegates.validations.AnalysisResult;
+import org.eclipse.lsp4j.*;
+import org.junit.jupiter.api.Test;
+
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/** Test to check SectionCompletion */
+class SectionCompletionTest {
+    private static final String TEXT =
+            "       Identification Division. \n"
+                    + "       Program-id.    ProgramId.\n"
+                    + "       Data Division.\n"
+                    + "       Working-Storage Section.\n"
+                    + "       01   outer1.\n"
+                    + "        02   INNER1      PIC 9(4) Binary. \n"
+                    + "        02   inner2      PIC X(10).\n"
+                    + "       Procedure Division section.\n"
+                    + "       000-Main-Logic Section.\n"
+                    + "           Perform 100-Test.\n"
+                    + "           Stop Run.\n"
+                    + "       100-Test Section.\n"
+                    + "           Move INNER1 of OUTER1 to Str.\n"
+                    + "       End program ProgramId.";
+
+    /**
+     * this test creates a test document, scans it for section names, adds them to completionItems,
+     * then attempts to find the expected output
+     */
+    @Test
+    void testSectionCompletion() {
+        CobolDocumentModel document = createModel();
+
+        Set<Completion> completionSet = new HashSet<>();
+        completionSet.add(new SectionCompletion());
+        Completions completions = new Completions(completionSet);
```

Comment:
new Completions(Set.of(new SectionCompletion()) so above two lines can be removed